controller/model/posture_check_model_os.go

```diff
@@ -26,6 +26,28 @@ type PostureCheckOperatingSystem struct {
 	OperatingSystems []OperatingSystem
 }
 
+func (p *PostureCheckOperatingSystem) Evaluate(pd *PostureData) bool {
+	if pd.Os.TimedOut {
+		return false
+	}
+
+	validOses := map[string]map[string]struct{}{}
```

Comment:
can validOses be done in an init or does it need to be done every call to Evaluate? It doesn't seem to use the posture data supplied which makes me think it might be eligible to be done via init instead?

Reply:
See the other comment! I replied in a different place.